docs/reference/stdlibs/std/testing.md

```diff
@@ -5,23 +5,23 @@ id: testing
 # Testing
 
 ```go
-func TestCurrentRealm() string
```

Comment:
Can you also remove this function and just use `std.CurrentRealm().PkgPath()`?

See `tests/stdlibs/std`

Reply:
Remove this from docs please